src/context/cartContext.js

```diff
@@ -0,0 +1,44 @@
+import { createContext, useContext, useReducer, useEffect } from "react";
+import { initialCartState, cartReducer } from "../redux";
+import { useAuth } from "../context";
+import { cartService } from "../services";
+
+const CartContext = createContext({
+  cartState: {},
+  cartDispatch: () => {},
+});
+
+const CartProvider = ({ children }) => {
+  const [cartState, cartDispatch] = useReducer(cartReducer, initialCartState);
+
+  const { authState } = useAuth();
+
+  useEffect(() => {
+    authState.token
+      ? (async () => {
+          try {
+            console.log("Authstate token in cartcontext", authState.token);
+            console.log("Useeffect called initially");
+            const response = await cartService(authState.token);
+            console.log(response.data.cart);
```

Comment:
Can remove the console.log's

Reply:
sure will do it. Maybe missed those somehow😬